Libraries/src/Amazon.Lambda.Serialization.SystemTextJson/DefaultLambdaJsonSerializer.cs

```diff
@@ -0,0 +1,159 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Text.Json;
+using Amazon.Lambda.Core;
+using Amazon.Lambda.Serialization.SystemTextJson.Converters;
+
+namespace Amazon.Lambda.Serialization.SystemTextJson
+{
+    /// <summary>
+    /// Custom ILambdaSerializer implementation which uses System.Text.Json
+    /// for serialization.
+    /// 
+    /// <para>
+    /// If the environment variable LAMBDA_NET_SERIALIZER_DEBUG is set to true the JSON coming
+    /// in from Lambda and being sent back to Lambda will be logged.
+    /// </para>
+    /// </summary>    
+    public class DefaultLambdaJsonSerializer : ILambdaSerializer
+    {
+        private const string DEBUG_ENVIRONMENT_VARIABLE_NAME = "LAMBDA_NET_SERIALIZER_DEBUG";
+        private readonly bool _debug;
+        
+        /// <summary>
+        /// The options used to serialize JSON object.
+        /// </summary>
+        protected JsonSerializerOptions SerializerOptions { get; }
+
+        /// <summary>
+        /// Constructs instance of serializer.
+        /// </summary>        
+        public DefaultLambdaJsonSerializer()
+        {
+            SerializerOptions = new JsonSerializerOptions()
+            {
+                IgnoreNullValues = true,
+                PropertyNameCaseInsensitive = true,
+                PropertyNamingPolicy = new AwsNamingPolicy(),
+                Converters =
+                {
+                    new DateTimeConverter(),
+                    new MemoryStreamConverter(),
+                    new ConstantClassConverter()
+                }
+            };
+
+            if (string.Equals(Environment.GetEnvironmentVariable(DEBUG_ENVIRONMENT_VARIABLE_NAME), "true", StringComparison.OrdinalIgnoreCase))
+            {
+                this._debug = true;
+            }            
```

Comment:
nit: should be ok to do this...

this._debug = string.Equals(Environment.GetEnvironmentVariable(DEBUG_ENVIRONMENT_VARIABLE_NAME), "true", StringComparison.OrdinalIgnoreCase);

Reply:
Done